docker/Dockerfile-ubuntu-20.04

```diff
@@ -8,33 +8,33 @@ RUN apt-get update \
         libglib2.0-dev libboost-dev libkrb5-dev libsystemd-dev libssl-dev \
         libboost-program-options-dev libboost-filesystem-dev byacc make libjsoncpp-dev
```

Comment:
Ubuntu 20.04 fails because it had clang 10 by default and does not support some of our flags declared in `CMakeLists.txt`. We can install and use clang 12 by adding:

```
RUN apt-get update \
    && DEBIAN_FRONTEND="noninteractive" TZ="${TIME_ZONE}" \
        apt install -y git clang wget curl autoconf \
        libglib2.0-dev libboost-dev libkrb5-dev libsystemd-dev libssl-dev \
        libboost-program-options-dev libboost-filesystem-dev byacc make libjsoncpp-dev \
        clang-12
    
RUN ln -sf /usr/lib/llvm-12/bin/clang /usr/bin/clang \
    && ln -sf /usr/lib/llvm-12/bin/clang++ /usr/bin/clang++
```

Reply:
Done